### PR TITLE
208

### PR DESCRIPTION
**Pull request by emillb**
Now users can see what they signed up for.

### CODE REVIEW

File: server/db/dummy/lectures.ts [hunk 1]
```diff
@@ -98,6 +98,28 @@ const lectures: DBLecture[] = [
     updatedBy: users[1].id,
     approved: true,
     draft: false,
+  },{
+    id: 'f546d5da-4128-4cc0-bf0e-b87c55e0b40e',
+    title: 'Är Java världens bästa programmeringsspråk?',
```
Comment (JonasKop) [inline server/db/dummy/lectures.ts:103]: Thou shalt not lie

File: webapp/src/section/eventViewer/EventRegistration.tsx [hunk 2]
```diff
@@ -73,11 +107,20 @@ const EventRegistration = ({
...
           justifyItems: 'center',
+          rowGap: '5px',
         }}
       >
         <Typography variant="h4">
-          {onCreate.isSuccess ? 'Tack för din anmälan!' : 'Du är redan anmäld till passet.'}
+          {onCreate.isSuccess ? 'Tack för din anmälan!' : 'Din anmälan'}
         </Typography>
+        {userAttendedLectures &&
+          userAttendedLectures.map((lectureColor: LectureInfo) => (
```
Comment (JonasKop) [inline webapp/src/section/eventViewer/EventRegistration.tsx:117]: Is `lectureColor` the wrong name?

**Review by JonasKop — CHANGES_REQUESTED** [review]

File: webapp/src/section/eventViewer/EventRegistration.tsx (hunk 2 above)
Comment (emillb) [reply webapp/src/section/eventViewer/EventRegistration.tsx:117]: Yesbox :D

File: server/db/dummy/lectures.ts (hunk 1 above)
Comment (emillb) [reply server/db/dummy/lectures.ts:103]: 🙈

**Review by JonasKop — APPROVED** [review]

**Pull request merged**